tools/generator/src/DTO/Platform.swift

```diff
@@ -47,6 +47,16 @@ extension Platform {
 \(arch)-apple-\(variant.triplePrefix)\(osVersion)\(variant.tripleSuffix)
 """
     }
+
+    mutating func merge(_ other: Platform) {
+        minimumOsVersion = other.minimumOsVersion
+    }
```

Comment:
Do we need a check to ensure that the platforms are mergeable? Can they only merge if they are compatible?

Reply:
The Starlark code that is determining target merges is only using direct dependencies, so we are trusting that is right.

Reply:
Given that, I think I can just take the platform entirely.